src/commands/build.ts

```diff
@@ -141,10 +141,22 @@ async function writeTypes (distDir: string, meta: ModuleMeta) {
     moduleImports.push('ModuleHooks')
     schemaShims.push('  interface NuxtHooks extends ModuleHooks {}')
   }
-  if (hasTypeExport('RuntimeModuleHooks')) {
-    moduleImports.push('RuntimeModuleHooks')
-    appShims.push('  interface RuntimeNuxtHooks extends RuntimeModuleHooks {}')
+
+  if (hasTypeExport('RuntimeModuleHooks') || hasTypeExport('ModuleRuntimeHooks')) {
+    const runtimeHooksInterfaces = []
+
+    if (hasTypeExport('RuntimeModuleHooks')) {
+      consola.warn('Please use \'ModuleRuntimeHooks\' instead of \'RuntimeModuleHooks\' as this misspelling will be removed in the future.')
```

Comment:
Nit, but I'd change the message to the one below so that it flows better (error → fix, rather than fix → error), is neutral about the previous naming choice, and utilises consola's ability to highlight `` `things` `` 🙂 
```suggestion
      consola.warn('`RuntimeModuleHooks` is a deprecated naming and will be removed in the future. Please use `ModuleRuntimeHooks` instead.')
```